ui-admin/src/HomePage.tsx

```diff
@@ -73,10 +73,10 @@ function HomePage() {
           </li>
         )}
       </ul>
-      { user.superuser && <Button variant='secondary' onClick={() => setShowNewStudyModal(true)}>
+      {user?.superuser && <Button variant='secondary' onClick={() => setShowNewStudyModal(true)}>
```

Comment:
nit: I think the convention is to have a space after curly braces in JSX, and it's consistent with the trailing space before the closing bracket

Reply:
Hmm, so I am actually struggling to get intellij to not remove the spaces. Doing some digging, it seems like most online docs / style guides recommend no spaces (https://react.dev/learn/tutorial-tic-tac-toe https://github.com/Shopify/javascript/tree/master/react#2.3 https://github.com/airbnb/javascript/tree/master/react#spacing). I don't feel super strongly but I can't get intellij to do it :'(